### DISCUSSION TITLE
ENTREGA PRACTICA HTML

### DISCUSSION THREAD

**Discussion by charlyguz** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#240] [Q&A] [answered]
Vi que las practicas de HTML se entregan el 4 pero no veo en donde se entrega, alguien sabe si hay link de forms o donde se entregan?

**Comment by doguedogue** [comment, marked as answer]
- Se va a entregar como limite hasta el domingo 6 (lo dijo hoy el MC)
- Mañana publican la forma  en donde las vamos a subir
- Felicidades si ya las acabaste!!

**Reply by charlyguz** [reply]
gracias

**Comment by JuanSalvador73** [comment]
Se entregan el domingo 6 - como dice doguedogue - y mañana publican la forma de entrega.

**Reply by JesusYG** [reply]
Hola, alguien sabe donde puedo ver los ejercicios de la Kata de HTML??? Intento entrar a traves del link de la Misión Frontend - JS, de la pagina "https://fernandaochoa.github.io/innovatemplate/project/missionfrontend", pero me sale error 404:
![Imagen1](https://user-images.githubusercontent.com/99446340/156850725-e421358f-f023-4633-8ab8-12b453dc73c2.png)

**Reply by JuanSalvador73** [reply]
Si tenía algún error, ya está arreglada:

![image](https://user-images.githubusercontent.com/99113565/156853749-63080296-078e-4887-abc6-4288d78dd71c.png)
